### PR TITLE
[IMP] web: show start time of regular event in calendar

### PR DESCRIPTION
**Pull request by pka-odoo**
Purpose of this commit is, to show  start time in month and week mode for
regular events (not multi-days).

After this commit:
For normal event(not multi-days), starting time of an event will be shown for
week and month mode in calendar.

task-2058477

Task: 
https://www.odoo.com/web?debug=assets#id=2058477&action=327&model=project.task&view_type=form&menu_id=4720

Pad:
https://pad.odoo.com/p/r.812305bded6dd5c39a632153a630a917

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/views/calendar/calendar_renderer.js [hunk 1]
```diff
@@ -422,15 +422,23 @@ return AbstractRenderer.extend({
...
                     var end = event.r_end || event.end;
                     // Detect if the event occurs in just one day
                     // note: add & remove 1 min to avoid issues with 00:00
                     var isSameDayEvent = start.clone().add(1, 'minute').isSame(end.clone().subtract(1, 'minute'), 'day');
-                    if (!event.record.allday && isSameDayEvent) {
-                        element.addClass('o_cw_nobg');
+                    if (isSameDayEvent && !self.hideTime) {
+                        if (event.showTime) {
+                            // Note: used fixed time format to display time in 12 hour format, we can
+                            // use time format from language configuration
```
Comment (aab-odoo) [inline addons/web/static/src/js/views/calendar/calendar_renderer.js]: i think we should...

File: addons/web/static/src/js/views/calendar/calendar_renderer.js [hunk 2]
```diff
@@ -422,15 +422,23 @@ return AbstractRenderer.extend({
...
+                            let displayTime = start.format('hh:mm A');
```
Comment (aab-odoo) [inline addons/web/static/src/js/views/calendar/calendar_renderer.js]: you can use `const` here

File: addons/web/static/src/js/views/calendar/calendar_renderer.js [hunk 3]
```diff
@@ -422,15 +422,23 @@ return AbstractRenderer.extend({
...
+                            element.find('.fc-content .fc-time').text(displayTime);
+                        }
+                        // For month view: Show background for all-day/multidate events only
+                        if (!event.record.allday) {
```
Comment (aab-odoo) [inline addons/web/static/src/js/views/calendar/calendar_renderer.js]: why do you do this only if hideTime is false? Not sure it's related...

File: addons/web/static/src/js/views/calendar/calendar_renderer.js (hunk 1 above)
Comment (aab-odoo) [reply addons/web/static/src/js/views/calendar/calendar_renderer.js]: I just checked with Eloise, and we should indeed use the default user formatting

Comment (pka-odoo) [reply addons/web/static/src/js/views/calendar/calendar_renderer.js]: Done :+1:

File: addons/web/static/src/js/views/calendar/calendar_renderer.js (hunk 2 above)
Comment (pka-odoo) [reply addons/web/static/src/js/views/calendar/calendar_renderer.js]: Done :+1:

File: addons/web/static/src/js/views/calendar/calendar_renderer.js (hunk 3 above)
Comment (pka-odoo) [reply addons/web/static/src/js/views/calendar/calendar_renderer.js]: @aab-odoo changes are Done :+1:

**Comment by aab-odoo** [comment]
robodoo r+ rebase-ff

**Pull request closed without merging**